src/commands/deploy/verifyAppSettings.ts

```diff
@@ -91,3 +98,45 @@ async function verifyWebContentSettings(context: IActionContext, remotePropertie
     context.telemetry.properties.webContentSettingsRemoved = String(shouldRemoveSettings);
     return shouldRemoveSettings;
 }
+
+function verifyLinuxRemoteBuildSettings(context: IActionContext, remoteProperties: { [propertyName: string]: string }, bools: VerifyAppSettingBooleans): boolean {
```

Comment:
Where did all of these extra settings come from? I don't think this was in the appservice package.

Reply:
Ah never mind, found the deleted file.  Thanks Eric 🥴